.github/workflows/update-tag.yml

```diff
@@ -0,0 +1,24 @@
+name: Update tag
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  update-tag:
+    permissions:
+      contents: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get major version num and update tag
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/}
+          MAJOR=${VERSION%%.*}
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Apologies but it isn't clear to me where this comes from.

Reply:
I'm not sure if we have a better way today though, all GitHub accounts have `<id>+<name>@users.noreply.github.com` email and `41898282` is the id for github-actions bot.
You can get the id from here https://api.github.com/users/github-actions[bot]

Here is a thread:

https://github.com/orgs/community/discussions/26560#discussioncomment-3252340